snow/networking/handler/metrics.go

```diff
@@ -37,9 +39,20 @@ func newMetrics(namespace string, reg prometheus.Registerer) (*metrics, error) {
 		Name:      "async_expired",
 		Help:      "Incoming async messages dropped because the message deadline expired",
 	})
+	// TODO: remove this comment
+	// This was in PlatformVM code before, and it was forced initialized to 0
+	// for each subnet. Moving this here means this will be initialized
+	// for per chain.
```

Comment:
When would we remove this comment? (I feel like this is really more of a comment on the PR than a comment for in the code). aka I think we can just remove this now.

Reply:
removed